VideoWeb/VideoWeb/ClientApp/src/app/shared/pipes/ngx-date.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { DatePipe } from '@angular/common';
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Pipe({
+    name: 'translateDate',
+    pure: false
+})
+export class NgxDatePipe implements PipeTransform {
+    constructor(private translateService: TranslateService) {}
+
+    public transform(value: any, pattern: string): any {
+        const datePipe = new DatePipe(this.translateService.currentLang);
+        return datePipe.transform(value, pattern);
```

Comment:
This needs to be tested. It probably passed the gate before because there was enough lines of new code that were touched by other tests to pass. Also again very minor but can we change pattern to format same as DatePipe.